app/views/home/quality.html.erb

```diff
@@ -1,34 +0,0 @@
-<% content_for :page_title do %>
-    Organisation quality -
```

Comment:
We shouldn't remove this.

Reply:
Similar to my comment on the dashboard - let's keep this. We'll have to work out access controls for some of these pages, which I assume is why it's come up now, but that can come later